homeassistant/components/rfxtrx/__init__.py

```diff
@@ -415,16 +358,8 @@ def unique_id(self):
         """Return unique identifier of remote device."""
         return self._unique_id
 
-    def turn_off(self, **kwargs):
-        """Turn the device off."""
-        self._send_command("turn_off")
-
-    def update_state(self, state, brightness=0):
-        """Update det state of the device."""
-        self._state = state
-        self._brightness = brightness
-        if self.added_to_hass:
-            self.schedule_update_ha_state()
+    def _apply_event(self, event):
```

Comment:
Maybe make this non private so we don't have to disable pylint?

Reply:
It should be private. (see other comment)

Reply:
Err. No other comment. Ok. I'll do that.